scripts/scr_unit_size/scr_unit_size.gml

```diff
@@ -1,23 +1,30 @@
-
-function scr_unit_size(armour, role, other_factors) {
+
+// lets face it this should all be solved in a method in the marines struct....
+function scr_unit_size(armour, role, other_factors, mobility) {
 
     // armour: armour
     // role: role
     // other_factors: +?
+	//mobility item
     var sz = 1;
     if (role!=""){
-
-        if (string_count("Dread",armour)>0) {sz=5;}
-        else if (role="Rhino") {sz=10;}
+		var bulky_armour = ["Terminator Armour", "Tartaros"]
+        if (string_count("Dread",armour)>0) {
+		sz+=5;
+	}else if (array_contains(bulky_armour,armour)){sz +=1};
+        if (role="Rhino") {sz=10;}

```

Comment:
the above 3 ifs are very hard to read, please change to something like:

```
        if (string_count("Dread",armour)>0) { sz+=5;}
	else if (array_contains(bulky_armour,armour)) {sz +=1;};
	// newline, because we are starting a new if chain below
        if (role="Rhino") {sz=10;}
```